src/components/DataList/tests/DataList.spec.ts

```diff
@@ -0,0 +1,194 @@
+import { describe, it, expect } from 'vitest'
+import { type DOMWrapper, mount } from '@vue/test-utils'
+import { vuetify } from '@tests/unit/setup'
+
+import DataList from '../DataList.vue'
+import { getDataList } from './fixtures'
+
+const items = getDataList()
+
+describe('DataList', () => {
+	it('renders correctly', () => {
+		const wrapper = mount(DataList, {
+			global: {
+				plugins: [vuetify],
+			},
+			propsData: {
```

Comment:
'propsData' est déprécié
https://test-utils.vuejs.org/migration/#propsdata-is-now-props